enterprise/app/trends/trends.css

```diff
@@ -77,6 +77,45 @@
   color: #82ca9d;
 }
 
+.drilldown-page-zoom-summary {
+  display: inline-block;
+  height: 40px;
+  margin: 0 0 0 36px;
+  vertical-align: bottom;
+}
+
+.drilldown-page-zoom-summary.buttonless {
+  margin: 0 0 0 61px;
```

Comment:
nit: the 61px looks slightly weird because it's not snapped to a multiple of 4px, maybe add a clarifying comment (or just snap to 4px if it looks ok)

Reply:
Added a comment--this is accounting for a border, so it's either this or a transparent border, so it'd need a comment no matter what :P